lms/envs/common.py

```diff
@@ -367,6 +367,15 @@
 
     # Show Language selector.
     'SHOW_LANGUAGE_SELECTOR': False,
+
+    # Write new CSM history to the extended table.
+    # Will eventually become the default and be deleted.
```

Comment:
Indeed - I'll clarify the comment.  Goal is that once CSMHE is deployed everywhere (sandboxes, devstack, prod, edge) everyone runs with 2 databases (this includes the eucalyptus opened release, and I've started talking to Ned about it).

Once that is true, we delete this feature flag from the code and all edx-platform installs have the second database.